packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -77,34 +77,82 @@ export function useSliderThumb(
   stateRef.current = state;
   let reverseX = direction === 'rtl';
   let currentPosition = useRef<number>(null);
+
+  let {keyboardProps} = useKeyboard({
+    onKeyDown(e) {
+      let {
+        getThumbMaxValue,
+        getThumbMinValue,
```

Comment:
Think the changes in this file are already on main? Does this branch need to be updated?

Reply:
color-area had not merged the changes from main.